include/stl2/view/all.hpp

```diff
@@ -46,7 +47,7 @@ STL2_OPEN_NAMESPACE {
 	// Work-around for https://gcc.gnu.org/bugzilla/show_bug.cgi?id=82507
 	template <class V, class R>
 	concept bool _ConstructibleFromRange =
-		View<V> && ext::ViewableRange<R> && Constructible<V, ext::all_view<R>>;
+		View<V> && Constructible<V, ext::all_view<R>>;
```

Comment:
This concept definition violates the "first instantiate types in a `typename` constraint" policy.